source/renderer/app/components/settings/categories/WalletsSettings.js

```diff
@@ -107,6 +110,9 @@ export default class WalletSettings extends Component<Props> {
               value={currencySelected ? currencySelected.symbol : null}
               options={currencyOptions}
               onChange={onSelectCurrency}
+              hasSearch={hasSearch}
+              skin={SelectSkin}
```

Comment:
this shouldn't be necessary as the skins are automatically mapped by react-polymorph (you only need to set it if you want to override with a specific skin)

Reply:
fixed